playbooks/cluster_setup_network.yaml

```diff
@@ -75,103 +134,62 @@
         - not apply_config
         - ovs_conf.changed
 
-- name: Configure PTP
```

Comment:
Also remove templates/00-ptp.netdev.j2 and templates/79-ptp.network.j2 and 79-ptp_interface.network.j2 files.
You will have to update the CI inventory to make that work.